test/e2e/passing/basic.expected

```diff
@@ -0,0 +1,8 @@
+Testing suite-name.
+This run has ID `00000000-0000-0000-0000-000000000000`.
+ ...                test-a          0   First test case.
[OK]                test-a          0   First test case.
+ ...                test-a          1   Second test case.
[OK]                test-a          1   Second test case.
+ ...                test-b          0   Third test case.
[OK]                test-b          0   Third test case.
+ ...                test-c          0   Fourth test case.
[OK]                test-c          0   Fourth test case.
+The full test results are available in `<build-context>/_build/_tests/00000000-0000-0000-0000-000000000000`.
+Test Successful in 0.000s. 4 tests run.
```

Comment:
Might be worth replacing the execution time by a placeholder as well as it is something likely to change for more complex tests!

Reply:
Nice catch. Done.